docs/ingest-and-test.md

```diff
@@ -71,8 +71,8 @@ python settings.py
 source env.sh
 ```
 
-Download [synthetic data](https://github.com/CanDIG/mohccn-data) and follow the instructions there.
+Follow the instructions for [Clinical data](https://github.com/CanDIG/candigv2-ingest#ingest-clinical-data) and [Genomic data](https://github.com/CanDIG/candigv2-ingest#ingest-genomic-files)
 
 ## Test data services
 
-Follow [Integration testing](https://candig.atlassian.net/wiki/spaces/CA/pages/665518081/Integration+testing) instructions to exercise your Docker installation.
\ No newline at end of file
+Follow [Integration testing](https://candig.atlassian.net/wiki/spaces/CA/pages/665518081/Integration+testing) instructions to exercise your Docker installation.
```

Comment:
we should include the instructions for running `make test-integration` to run the automated test suite, and then also link to the confluence page (but only if it is public)